makefile.go

```diff
@@ -0,0 +1,220 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+)
+
+var (
+	KUBE_GO_PACKAGE string
+	K8SM_GO_PACKAGE string
+	K8_CMD          []string
+	tempDirs        []string
+	CMD_DIRS        []string
+	LIB_DIRS        []string
+	target          = "./target"
+)
+
+func run(dir, command string, args ...string) (output string, err error) {
+	cmd := exec.Command(command, args...)
+	cmd.Dir = dir
+
+	var out bytes.Buffer
+	var stderr bytes.Buffer
+	cmd.Stdout = &out
```

Comment:
why capture the output/error streams instead of just configuring cmd to use `os.Stdout` and `os.Stdin`?